src/main/java/org/maxgamer/quickshop/shop/ContainerShop.java

```diff
@@ -401,19 +428,30 @@ public void delete(boolean memoryOnly) {
         if (memoryOnly) {
             // Delete it from memory
             plugin.getShopManager().removeShop(this);
+            onUnload();
```

Comment:
Why we need to call `onUnload()` for double times?

Reply:
That was a remnant of my extensive checking to make sure it unloaded the duplicates I had made via other methods. Good catch!